README.md

```diff
@@ -184,21 +184,21 @@ ssh -i emma.key root@emma2.$EMMA_DOMAIN uptime
 Create the `hosts` file see `hosts.template` for template.
 Now use ansible to verify login.
 ```
-ansible all --private-key=emma.key -u root -i hosts -m ping
+ansible all --private-key=emma.key -u root --ssh-extra-args="-o StrictHostKeyChecking=no" -i hosts -m ping
```

Comment:
I think it is a good idea. 
However, I need to read on how to do it. The best is to open an issue requesting such enhancement.